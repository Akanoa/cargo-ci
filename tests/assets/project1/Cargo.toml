[package]
name = "project1"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
[dependencies]
toto = {git = "git@gitlab.com:orga/project.git", tag="1.0.0"}

[dev-dependencies.titi]
git = "git@gitlab.domain.tld:myorg/sub_project/too/deep.git"
tag = "1.0.0"
default-no-features = "true"
features = [
 "feature1",
 "test"
]