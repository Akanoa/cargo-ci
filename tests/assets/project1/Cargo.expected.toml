[dependencies.toto]
git = "https://gitlab-ci-token:token123@gitlab.com/orga/project.git"
tag = "1.0.0"
[dev-dependencies.titi]
default-no-features = "true"
features = ["feature1", "test"]
git = "https://gitlab-ci-token:token123@gitlab.domain.tld/myorg/sub_project/too/deep.git"
tag = "1.0.0"

[package]
edition = "2021"
name = "project1"
version = "0.1.0"
